# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/tumbler/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- tumbler-4.18.2/plugins/jpeg-thumbnailer/jpeg-thumbnailer.c.vanilla	2024-07-23 13:26:51.511311984 +0200
+++ tumbler-4.18.2/plugins/jpeg-thumbnailer/jpeg-thumbnailer.c	2024-07-23 13:27:32.589308026 +0200
@@ -165,7 +165,7 @@
 
 
 
-static gboolean
+static boolean
 tvtj_fill_input_buffer (j_decompress_ptr cinfo)
 {
   struct jpeg_source_mgr *source = cinfo->src;
